sunpy/net/base_client.py

```diff
@@ -85,6 +85,25 @@ def __repr__(self):
     def _repr_html_(self):
         return self.build_table()._repr_html_()
 
+    def show(self, *cols):
+        """
+        Returns response tables with desired columns for the Query.
+
+        Parameters
+        ----------
+        \\*cols : `tuple`
```

Comment:
Is this not a list?

Reply:
Method is expected to be used as
`q.show('Start Time', 'Instrument', 'Source')`
not as 
`q.show(['Start Time', 'Instrument', 'Source'])`
so it is a tuple since *args passed to python method are always tuple.

Reply:
But that isn't a tuple either?

Reply:
Hmm, user specify column names as args to the method.
However `type(cols)` do give a tuple.